recipes/libharu/all/conanfile.py

```diff
@@ -1,16 +1,23 @@
-from conan.tools.microsoft import is_msvc
-from conans import CMake, ConanFile, tools
-import functools
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.microsoft import check_min_vs, is_msvc_static_runtime, is_msvc
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, copy, rm, rmdir, replace_in_file, load, save
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.env import VirtualBuildEnv
 import os
 import re
 
-required_conan_version = ">=1.45.0"
+
+required_conan_version = ">=1.53.0"
+
 
 
 class LibharuConan(ConanFile):
     name = "libharu"
     description = "Haru is a free, cross platform, open-sourced software library for generating PDF."
-    topics = ("libharu", "pdf", "generate", "generator")
+    topics = "pdf", "generate", "generator"
     license = "ZLIB"
```

Comment:
Based on https://spdx.org/licenses/ only first letter should be uppercase
```suggestion
    license = "Zlib"
```